### DISCUSSION TITLE
Make clear optional

### DISCUSSION THREAD

**Discussion by srijan** [shinsenter/php#32] [Ideas]
When the container starts, it clears the console and prints the welcome message.

While the welcome message is customizable, the clearing is not. I want to skip clearing the console because it sometimes hides other messages I want to see.

Maybe there's already a way to do this, but I could not find it.

**Comment by srijan** [comment]
Right now, I'm working around this by mounting my own copy of `/etc/cont-init.d/999-fix-permission` with `clear` removed.

**Comment by shinsenter** [comment]
@srijan 
Thank you for the feedback.
I think the clear command isn't very useful, so I removed it from the `999-fix-permission` file.
